Index: src/main/java/org/example/RoundValues.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/org/example/RoundValues.java b/src/main/java/org/example/RoundValues.java
new file mode 100644
--- /dev/null	(date 1702495228219)
+++ b/src/main/java/org/example/RoundValues.java	(date 1702495228219)
@@ -0,0 +1,4 @@
+package org.example;
+
+public class RoundValues {
+}
